go.mod

```diff
@@ -28,8 +28,8 @@ require (
 	github.com/lightstep/lightstep-tracer-go v0.26.0
 	github.com/miekg/dns v1.1.56
 	github.com/oklog/ulid v1.3.1
-	github.com/open-policy-agent/opa v0.58.0
-	github.com/open-policy-agent/opa-envoy-plugin v0.58.0-envoy
+	github.com/open-policy-agent/opa v0.54.0
```

Comment:
For some reason it downgrades other dependencies (I think this might be related to https://github.com/golang/go/issues/60588)
I guess we would need to upgrade golang.org/x/net manually.

Reply:
The last time I bundled all dependency updates and it passed also the tests in the first run. Not sure why it fails again consistently for dependency updates...